carte_cli/extractor/glue_extractor.py

```diff
@@ -41,6 +42,40 @@ def _get_column_type(self, column: Dict) -> str:
         else:
             return col_type
 
+    def _get_glue_table_columns(
+        self, row: Dict[str, Any], table_name: str
+    ) -> List[ColumnMetadata]:
+        columns = []
+        if "spark.sql.sources.schema" in row["Parameters"]:
+            # For delta and parquet tables if the schema is not too big
+            schema = json.loads(row["Parameters"]["spark.sql.sources.schema"])
+        elif "spark.sql.sources.schema.numParts" in row["Parameters"]:
+            # If the delta or parquet table's schema is very big, glue separates it to multiple properties ¯\_(ツ)_/¯
+            schema_parts_count = int(
+                row["Parameters"]["spark.sql.sources.schema.numParts"]
+            )
+            schema_str = ""
+            for part in range(0, schema_parts_count):
+                schema_str = (
+                    schema_str
+                    + row["Parameters"][f"spark.sql.sources.schema.part.{part}"]
+                )
+            schema = json.loads(schema_str)
+        else:
+            print(table_name)
+            print(row)
+            raise Exception("Unsupported glue table format")
```

Comment:
You probably shouldn't print this without explanation, but rather make the table and row part of the exception's message